### PR TITLE
add node_metrics id

### PR DESCRIPTION
**Pull request by laipogo**

### CODE REVIEW

File: protos/node/node.proto [hunk 1]
```diff
@@ -35,37 +35,38 @@ message Node {
...
-  uint64 memory_used = 7;
-  double cpu_used = 8;
-  uint64 upload = 9;
-  uint64 download = 10;
-  uint64 total_upload = 11;
-  uint64 total_download = 12;
-  uint64 storage_price_deal = 13;
-  uint64 bandwidth_price_deal = 14;
-  Settings settings = 15 [
+  uint64 id = 2;
```
Comment (taiyangc) [inline protos/node/node.proto]: append at the end, do not change position of everything

Comment (laipogo) [reply protos/node/node.proto]: i c.

Comment (taiyangc) [reply protos/node/node.proto]: maybe we should also add a `json:"-"` to this field since it should *never* be exposed to public, thoughts @wxue ?

Comment (taiyangc) [reply protos/node/node.proto]: I mean, a specialized field that is not defined as a valid protobuf field, but only generated for pg tags

**Pull request closed without merging**